src/Microsoft.Management.Deployment/PackageManager.cpp

```diff
@@ -39,6 +39,15 @@ using namespace ::AppInstaller::CLI::Execution;
 
 namespace winrt::Microsoft::Management::Deployment::implementation
 {
+    PackageManager::PackageManager()
+    {
+        auto previousThreadGlobals = m_threadGlobals.SetForCurrentThread();
+        // Immediately reset as we only want the thread globals for logging within this object.
+        previousThreadGlobals.reset();
+        m_threadGlobals.GetTelemetryLogger().SetCaller(GetCallerName());

```

Comment:
Make sure that this *doesn't* send a summary event.  There is a bool that disables it as I remember.